media-gfx/blender/blender-2.83.17.ebuild

```diff
@@ -0,0 +1,380 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
Python 3.7 isn't supported by the eclass anymore, so we can drop this (it's a noop).